server/app/controllers/application_controller.rb

```diff
@@ -1,4 +1,21 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::API
+  before_action :set_locale
+
+  private
+    def set_locale
+      # locale = locale_from_url || I18n.default_locale
+      # use this with routes.rb: scope "(:locale)", locale /#{I18n.available_locales.join("|")}/ do
+      I18n.locale = request.headers["X-Lang"] || I18n.default_locale
```

Comment:
it would be good to have a way to switch language via URL or user info, so that language is not limited to what browser uses...